docs/zh/latest/plugins/node-status.md

```diff
@@ -40,7 +40,7 @@ title: node-status
 
 ## 插件接口
 
-插件增加接口 `/apisix/status`，可通过 [interceptors](../plugin-interceptors.md) 保护该接口。
+插件增加接口 `/apisix/status`，需要通过 [public-api](public-api.md) 插件来暴露它。
```

Comment:
Let's update the link to point to English version like https://github.com/apache/apisix/blob/48e8a1ee483caa7150f7ad812953730eb50324bb/docs/zh/latest/plugins/kafka-logger.md?plain=1#L183

Reply:
Hi, @spacewander.

As currently used, it renders properly on our official website (when no language-specific localization documents are returned to English), and if we modify it directly, it will cause confusion on the official website, i.e. some multilingual documents will be all linked back to the English version.
It seems like we can't have it both ways, but I think it's more important to make sure the content on the official website is always available in the right way than to use the md file to read it directly.

Currently there is no Chinese documentation for the `public-api` plugin, if you think it is really necessary to change it, I will do so.

Reply:
How about using English only in this PR? After it gets merged, @guoqqqi and @Baoyuantop could help to translate 🤔

Reply:
@bzp2010 @juzhiyuan 
We already do this in other loggers. Although someone else can translate it later, but we should not invent another way to handle it, and merge a PR which will create known broken links.

Reply:
@spacewander updated